src/plugins/recordTypes/citation/fields.js

```diff
@@ -0,0 +1,51 @@
+export default (configContext) => {
+  const {
+    TextInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  const {
+    extensions,
+  } = configContext.config;
+
+  return {
+    document: {
+      'ns2:citations_common': {
+        citationTermGroupList: {
+          citationTermGroup: {
+            termSource: {
+              [config]: {
+                view: {
+                  type: TextInput,
+                },
+              },
+            },
+          },
+        },
+        citationRelatedTermsGroupList: {
+          citationRelatedTermsGroup: {
+            relatedTerm: {
+              [config]: {
+                view: {
+                  type: TextInput,
+                },
+              },
+            },
+          },
+        },
+        citationNote: {
+          [config]: {
+            view: {
+              props: {
+                multiline: false,
```

Comment:
Don't need this. It was not customized in 4.1. If something changed in the core system between 4.1 and 5.0, in general just inherit it. Don't try to undo the change.